app/controllers/v1/cloud9_controller.rb

```diff
@@ -1,30 +1,17 @@
 module V1
   class Cloud9Controller < ApplicationController
-    EMAIL_REGEX = /\A([^@\s]+)@((?:[-a-z0-9]+\.)+[a-z]{2,})\z/i
-
     def send_invite
-      unless EMAIL_REGEX.match(email)
-        return render json: { errors: { email: 'Must be a valid email' } },
-                      status: 422
-      end
-
-      invite_member(email)
+      invite = Cloud9Invite.new(email: email)
 
-      render json: { success: true }
-    rescue RestClient::Conflict
-      render json: { errors: { email: 'Invite already sent for this email' } },
-             status: 409
+      if invite.save
+        render json: { success: true }
```

Comment:
I didn't mention this before, but since this endpoint is now becoming more RESTful, we should probably return the created `Cloud9Invite` object here with a 201 status code.

We should probably also move the endpoint to `POST /cloud9/invites`